rust/src/combinators/select_private_candidate/make_select_private_candidate.tex

```diff
@@ -0,0 +1,104 @@
+\documentclass{article}
+\input{../../lib.sty}
+
+\title{\texttt{fn make\_select\_private\_candidate}}
+\author{Michael Shoemate}
+\date{}
+
+\begin{document}
+
+\maketitle
+
+\contrib
+Proves soundness of \rustdoc{combinators/fn}{make\_select\_private\_candidate} in \asOfCommit{mod.rs}{f5bb719}.
+This constructor is a special case of \rustdoc{combinators/fn}{make\_select\_private\_candidate}.
+The proof for this constructor appeals to the proof for \rustdoc{combinators/fn}{make\_select\_private\_candidate}.
```

Comment:
I'm confused here: It seems like the thing that you're proving is also what this is a special case of?